build.gradle.kts

```diff
@@ -130,6 +130,9 @@ dependencies {
 
     integrationTestImplementation(libs.junit.jupiter)
     integrationTestImplementation(libs.logback.classic)
+    integrationTestImplementation(libs.hibernate.testing) {
+        exclude(group = "org.apache.logging.log4j", module = "log4j-core")
+    }
```

Comment:
`hibernate-testing` relies on log4j as logging option conflicting with logback. Disabling it makes our logging take effect.